### DISCUSSION TITLE
Any idea of adding a new event calander component ? 🎉

### DISCUSSION THREAD

**Discussion by jameswong3388** [shadcn-ui/ui#2300] [Ideas]
A calander component but can interact with events, like [react-event-calendar](https://www.npmjs.com/package/react-event-calendar)

**Comment by DRakkola** [comment]
yeah it would be a nice addition

**Comment by birobirobiro** [comment]
https://github.com/jquense/react-big-calendar

It`s a great ideia

**Comment by madanbajagai3** [comment]
This would be great addition.

**Comment by b2rsp** [comment]
This would be awesome!

**Comment by dir** [comment]
+1 to this, would be an absolutely incredible component to add to the shadcn suite. Hard though, he usually builds on top of existing libraries and most existing libraries are pretty rough.

**Comment by list-jonas** [comment]
Hey everyone!

I saw the interest in having a Shadcn-themed event calendar component, and I’m excited to share something I’ve been working on. I just created a theme for **React Big Calendar** that is fully integrated with **Shadcn** styles.

You can check out the repo here: 
[https://github.com/list-jonas/shadcn-ui-big-calendar](https://github.com/list-jonas/shadcn-ui-big-calendar)

Contributions and feedback are more than welcome!

Looking forward to seeing what you all think!

**Reply by njoshi22** [reply]
I actually came across this on google a few days ago and considered building off it. Great work!

**Reply by dir** [reply]
I'll have to check this out right away! Thanks for making this.
